client/my-sites/site-settings/delete-site/index.jsx

```diff
@@ -263,14 +260,17 @@ class DeleteSite extends Component {
 								<li className="delete-site__content-list-item">
 									{ translate( 'Purchased Upgrades' ) }
 								</li>
+								<li className="delete-site__content-list-item">
+									{ translate( 'Premium Themes' ) }
+								</li>
 							</ul>
 						</ActionPanelFigure>
 						{ ! isAtomic && (
 							<div>
 								<p>
 									{ translate(
 										'Deletion {{strong}}can not{{/strong}} be undone, ' +
```

Comment:
Hi! I've found a possible matching string that has already been translated **27** times:
`translate( 'Deletion {{strong}}can not{{/strong}} be undone, and will remove all content, contributors, domains, and upgrades from this site.' )` *ES Score: 12*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

Reply:
`translate` can handle strings split with the concatenation operator `+`, but it looks like the bot can't :)